tmt/steps/provision/__init__.py

```diff
@@ -42,233 +42,6 @@ class CheckRsyncOutcome(enum.Enum):
     INSTALLED = 'installed'
 
 
-class ProvisionPlugin(tmt.steps.GuestlessPlugin):
```

Comment:
This looks like a failed rebase, I certainly would not expect so much code to be moved around. `ProvisionPlugin` class has been recently moved in front of `Provision` class, because `Provision._base_plugin_class` points to `ProvisionPlugin`.

Reply:
Hm.. I moved Guest class on front because I need the type Guest to be known in class ProvisionPlugin(tmt.steps.GuestlessPlugin).

Reply:
Aaah, that's it, thanks.